src/components/pathway-progress/SubscriptionStatusCard.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useContext } from 'react';
+import { Card, Badge } from '@edx/paragon';
+
+import { UserSubsidyContext } from '../enterprise-user-subsidy';
+import { LICENSE_STATUS } from '../enterprise-user-subsidy/data/constants';
+import { SubsidyRequestsContext, SUBSIDY_TYPE } from '../enterprise-subsidy-requests';
+
+function SubscriptionStatusCard() {
```

Comment:
Can we reuse this existing component here? https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/master/src/components/dashboard/sidebar/SubscriptionSummaryCard.jsx

Reply:
That component adds information that as pre the mockups is not needed by here, that is why I created a new component.